integration/setup/common.go

```diff
@@ -93,10 +94,13 @@ func SkipForPostgresWithReason(tb testing.TB, reason string) {
 
 // setupListener starts in-process FerretDB server that runs until ctx is done,
 // and returns listening port number.
-func setupListener(tb testing.TB, ctx context.Context, logger *zap.Logger) int {
+func setupListener(tb testing.TB, ctx context.Context, logger *zap.Logger) (*state.Provider, int) {
 	tb.Helper()
 
-	p, err := state.NewProvider("")
+	stateFile, err := os.CreateTemp("", "test-*.state")
+	require.NoError(tb, err)
+
+	p, err := state.NewProvider(stateFile.Name())
```

Comment:
```suggestion
	p, err := state.NewProvider("")
```

We don't have to persist anything in those tests

Reply:
@AlekSi but will it work? I mean, will I be able to change the state through `Update` and expect it to be persistent if there is no state file?

Reply:
Yes, there are tests for that